cluster/common/velero/main.tf

```diff
@@ -1,5 +1,5 @@
 module "common-provider" {
-  source = "github.com/microsoft/bedrock?ref=master//cluster/common/provider"
+  source = "github.com/microsoft/bedrock?ref=byo.rg//cluster/common/provider"
```

Comment:
Will we need to switch this back to master as the source?

Reply:
Yes